Components/GamesDisplay/index.js

```diff
@@ -0,0 +1,161 @@
+        (game) =>
+          (game.home_team.id === parseID &&
+            game.home_team_score < game.visitor_team_score) ||
+          (game.visitor_team.id === parseID &&
+            game.visitor_team_score < game.home_team_score)
+      )
+    : regularseasonGames.filter(
+        (game) =>
+          (game.home_team.id === parseID &&
+            game.home_team_score < game.visitor_team_score) ||
+          (game.visitor_team.id === parseID &&
+            game.visitor_team_score < game.home_team_score)
+      );
+
+  let seasons = [];
+  for (let year = CURRENT_SEASON; year >= 1946; year--) {
+    seasons.push(year);
+  }
+  const selectYears = seasons.map((season) => ({
+    value: season,
+    label: season.toString(),
+  }));
+
+  function handleSeasonChange(selectedOption) {
+    setSeason(selectedOption.value);
+  }
+
+  function toggleView() {
+    setShowPostseason((prevState) => !prevState);
+  }
+
+  return (
+    <>
+      {filteredGames.length > 0 ? (
+        <GamesContainer>
+          <SelectionContainer>
+            <Headline>Games</Headline>
+            <label htmlFor="season-select">Season:</label>
+            <StyledSelect
+              value={selectYears.find((option) => option.value === season)}
+              options={selectYears}
+              onChange={handleSeasonChange}
+              placeholder="pick a season"
+              id="season-select"
+            />
+            <h3>{showPostseason ? "Postseason" : "Regular Season"}</h3>
+            <h4>Wins: {gamesWon.length}</h4>
+            <h4>Losses: {gamesLost.length}</h4>
+            <button type="button" onClick={toggleView}>
+              {showPostseason ? "Show Regular Season" : "Show Postseason"}
+            </button>
+          </SelectionContainer>
+          {showPostseason ? (
+            postseasonGames.length > 0 ? (
+              postseasonGames.map((game) => (
+                <GamesList key={game.id}>
+                  <SingleGame key={game.id}>
+                    {game.date.split("T")[0]}
+                    {": "}
+                    <br />
+                    <br />
+                    {game.home_team.full_name} {game.home_team_score} -{" "}
+                    {game.visitor_team_score} {game.visitor_team.full_name}
+                  </SingleGame>
+                </GamesList>
+              ))
+            ) : (
+              <NoData>No Playoff Games for selected season</NoData>
+            )
+          ) : regularseasonGames.length > 0 ? (
+            regularseasonGames.map((game) => (
+              <GamesList key={game.id}>
+                <SingleGame key={game.id}>
+                  {game.date.split("T")[0]}
+                  {": "}
+                  <br />
+                  <br />
+                  {game.home_team.full_name} {game.home_team_score} -{" "}
+                  {game.visitor_team_score} {game.visitor_team.full_name}
+                </SingleGame>
+              </GamesList>
```

Comment:
```
              <GamesList key={game.id}>
                <SingleGame key={game.id}>
                  {game.date.split("T")[0]}
                  {": "}
                  <br />
                  <br />
                  {game.home_team.full_name} {game.home_team_score} -{" "}
                  {game.visitor_team_score} {game.visitor_team.full_name}
                </SingleGame>
              </GamesList>
```

You could think about changing this part into a separate component as you use it more than once in this component. This can reduce the length of this file. 😅

Reply:
> Hi Moritz, great job! :octocat: I left some reamrks for you to consider, but this does not stop me from approving your PR. :)

Thanks for looking thoroughly through my code, not just the parts that I changed. All of your advice is much appreciated. I've adapted my code accordingly, except for the component part. That's most likely a ToDo for tmrw. 😃